source/jormungandr/jormungandr/instance.py

```diff
@@ -655,34 +665,26 @@ def init(self):
         return False
 
     def get_street_network(self, mode, request):
-        return self._streetnetwork_backend_manager.get_street_network(self, mode, request)
+
+        if app.config[str('DISABLE_DATABASE')]:
+            return self._streetnetwork_backend_manager.get_street_network_legacy(self, mode, request)
+        else:
+            column_in_db = fallback_modes.street_network_modes_in_db.get(mode, None)
+            streetnetwork_backend_conf = getattr(self, column_in_db)
+            return self._streetnetwork_backend_manager.get_street_network_db(self, streetnetwork_backend_conf)
```

Comment:
Here, you have to get the name of the column (ex: sn_car), then the value that's in this column (ex: valhalla) and then get in the sn table in the db, the config of the sn. Is that it?

Reply:
Yes, that's exactly that!